trendmicro_deepsecurity/Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM komand/python-3-37-plugin:3
+#FROM komand/python-3-37-plugin:3
```

Comment:
No need to keep this line, it can be removed :)

Reply:
done